js/projectsConstructor.js

```diff
@@ -9,22 +9,48 @@ function Project (projectsListObj) {
   Project.all.push(this);
 }
 
-Project.prototype.renderProject = function() {
+Project.prototype.toHtml = function() {
   var $newProject = $('.template').clone();
   $newProject.removeClass('template');
   $newProject.find('h3')
-             .text(this.name);
+             .text(this.title);
   $newProject.find('a')
-             .eq(1).attr('href', 'this.url');
+             .eq(0).attr('href', this.url);
   $newProject.find('a')
-             .eq(2).attr('href','this.githubURL');
+             .eq(1).attr('href',this.githubURL);
   $newProject.find('img')
-             .eq(2).src('this.img');
+             .attr('src',this.img);
   $newProject.find('p')
-             .eq(1).text('this.skillsUsed');
+             .eq(0).text(this.skillsUsed);
+  $('.anchorProjects').append($newProject);
+};
 
-  $('.template').append($newProject);
+Project.all = [];
 
-}
+projectsList.forEach(function(project) {
+  new Project(project);
+});
 
-Project.all = [];
+Project.all.forEach(function(project) {
+  project.toHtml();
+});
+
+$('#aboutMe').on('click', function(){
+  $('main').hide();
```

Comment:
Since you do not show any of the elements in any of the other event handlers, if you click the 'About' button you can never see the rest of the page ever again.